docs/product-assembly_designing-and-building-cases/src/main/asciidoc/case-management-security-configuration-proc.adoc

```diff
@@ -0,0 +1,58 @@
+[id='case-management-security-configuration-proc-{context}']
+= Configuring security for case management
+
+You can turn off case instance authorization by setting the following system property to `false`:
+
+`org.jbpm.cases.auth.enabled`
+
+This system property is just one of the security components for case instances.In addition, you can configure case operations at the execution server-level using a property file called `case-authorization.properties`, which is available at the root of the class path when you start a case application. 
```

Comment:
I would remove this part

> when you start a case application

and put there "of the execution server application (kie-server.war/WEB-INF/classes)"